Jellyfin.Server/StartupOptions.cs

```diff
@@ -0,0 +1,41 @@
+using CommandLine;
+using Emby.Server.Implementations;
+
+namespace Jellyfin.Server
+{
+    /// <summary>
+    /// Class used by CommandLine package when parsing the command line arguments.
+    /// </summary>
+    public class StartupOptions : IStartupOptions
+    {
+        [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
+        public string DataDir { get; set; }
+
+        [Option('c', "configdir", Required = false, HelpText = "Path to use for config data (user policies and puctures).")]
+        public string ConfigDir { get; set; }
+
+        [Option('l', "logdir", Required = false, HelpText = "Path to use for writing log files.")]
+        public string LogDir { get; set; }
+
+        [Option("ffmpeg", Required = false, HelpText = "Path to external FFmpeg exe to use in place of built-in.")]
```

Comment:
```suggestion
        [Option("ffmpeg", Required = false, HelpText = "Path to external FFmpeg executable to use in place of default found in PATH.")]
```
or something similar... we don't have a built-in `ffmpeg`.

Reply:
Ok